calico_containers/pycalico/ipam.py

```diff
@@ -33,7 +33,7 @@
 
 _log = logging.getLogger(__name__)
 
-IPAM_V_PATH = "/calico/ipam/v1/"
+IPAM_V_PATH = "/calico/ipam/v2/"
```

Comment:
It's a lot of work.  Off the cuff, I'd guess at least a developer-month by the time we design, implement, UT and ST.